### PR TITLE
fix: kwin-system76-scheduler-integration

### PR DESCRIPTION
**Pull request by madonuko**
Added script to enable the kwin script automatically

### CODE REVIEW

**Review by lleyton — CHANGES_REQUESTED** [review]
Doesn't work, we already try something like this in Ultramarine, similarly modifying the /usr/sharekde-settings/kde-profile/default/share/config/kwinrc path, with no success.

File: anda/others/kwin-system76-scheduler-integration/enable-kwin-system76-scheduler-integration.service [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Enable kwin-system76-scheduler-integration
+After=network.target
+
+[Service]
+Type=simple
```
Comment (korewaChino) [inline anda/others/kwin-system76-scheduler-integration/enable-kwin-system76-scheduler-integration.service]: use oneshot service

**Review by korewaChino — APPROVED** [review]

**Review by korewaChino — DISMISSED** [review]

**Review by korewaChino — APPROVED** [review]
hacky, but should work

**Comment by korewaChino** [comment]
superceded by https://github.com/Ultramarine-Linux/ultramarine-pkgs/commit/7882ee51e67bde081eabc89075c3fc8e71817d3d

**Pull request closed without merging**